includes/Integrations/Jetpack.php

```diff
@@ -68,6 +83,76 @@ public function add_to_jetpack_sitemap( $post_types ) {
 		return $post_types;
 	}
 
+	/**
+	 * Add VideoPress to allowed mime types. If the site does not support videopress, this allow will be filtered out.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @param array $mime_types Associative array of allowed mime types per media type (image, audio, video).
+	 *
+	 * @return array
+	 */
+	public function add_videopress( array $mime_types ) {
+		$mime_types['video'][] = self::MIME_TYPE;
```

Comment:
Is there any harm in not adding a `Jetpack_Plan::supports( 'videopress' )` check?

Reply:
What would be the benefit?

Reply:
My issue with using a class from another plugin, is we have to check it exists first. Also this class is removed and changed, it break things for us...

Reply:
Yeah it would need safety guards of course. (Alternatively could be contributed to Jetpack instead)

I am asking because a) you had that in your example code snippet on the ticket and b) there could be issues if support is added but the module isn't actually active on the site. And I want to avoid such issues if there are any :-)

For some reason Jetpack checks the availability in the Gutenberg integration as well:

https://github.com/Automattic/jetpack/blob/86e408280b255d78f69b3584d1ab2e14b49d9d26/projects/plugins/jetpack/modules/videopress/class.videopress-gutenberg.php#L61-L136

Reply:
Why don't we merge this as is now. I will create a PR on jetpack's side. Once that is merged, I will remove this line. Is that fair?

Reply:
Alright, let's leave as-is and then see how it works before proceeding further.